sdk/python/tests/compiler/testdata/coin.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2018 Google LLC
```

Comment:
For this test, I meant we should use this to replace the current [condition.py](https://github.com/kubeflow/kfp-tekton/blob/master/sdk/python/tests/compiler/testdata/condition.py) test. Also, can you add a note under the https://github.com/kubeflow/kfp-tekton/tree/master/sdk#tested-versions to warn users that they need master branch if they are using condition with parameters. Thanks.